Watchman.DomainModel/Mute/ProtectionPunishmentOption.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace Watchman.DomainModel.Protection
+﻿namespace Watchman.DomainModel.Mute
```

Comment:
Mute to niezbyt dobra nazwa 

bardziej w roles powinna być opcja dodania roli muted 
albo w users opcja zmutowania użytkownika

Reply:
Zmienić z powrotem na Protection?

Reply:
myśle czy Protection będzie odpowiednie 

myśle że to by bardziej pasowało do Users

Reply:
Wszystko co jest w Mute scalić z Users?

Reply:
myśle że tak 
ostatecznie dobrze byłoby, gdyby mutowanie mogło iść z poziomu użytkownika
ale to już zrobimy osobno podczas refaktoryzacji projektu 
na razie skończmy tą funkcjonalność tak, żeby działała prawidłowo